setup.cfg

```diff
@@ -43,8 +43,8 @@ python_requires = >=3.5
 
 [options.entry_points]
 console_scripts =
-	pytest=pytest:main
-	py.test=pytest:main
+	pytest=pytest:console_main
```

Comment:
It feels wrong to me to use a private function as the endpoint. I guess I just want the entry point to `pytest` to be in `pytest` :smile: 

Since I don't expect this would ever need to be changed (the signature being strictly specified), do you mind if I keep it as is?

BTW, I was originally thinking the wrapper should just be straight in `pytest/__main__.py`, but decided against it since we don't want any functional code there.

Reply:
> Since I don't expect this would ever need to be changed (the signature being strictly specified)

I'm not worried about the signature changing, is just that being public API we need to keep it around even if we don't need it anymore in the future for example. But OK, I don't think this is a big deal.

> do you mind if I keep it as is?

Sure, I don't feel strongly about it. 👍